{
    "GSD": {
        "alias": "CVE-2022-30613",
        "id": "GSD-2022-30613"
    },
    "namespaces": {
        "cve.org": {
            "CVE_data_meta": {
                "ASSIGNER": "cve@mitre.org",
                "ID": "CVE-2022-30613",
                "STATE": "RESERVED"
            },
            "data_format": "MITRE",
            "data_type": "CVE",
            "data_version": "4.0",
            "description": {
                "description_data": [
                    {
                        "lang": "eng",
                        "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
                    }
                ]
            }
        },
        "nvd.nist.gov": {
            "configurations": {
                "CVE_data_version": "4.0",
                "nodes": []
            },
            "cve": {
                "CVE_data_meta": {
                    "ASSIGNER": "psirt@us.ibm.com",
                    "ID": "CVE-2022-30613"
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "en",
                            "value": "IBM QRadar SIEM 7.4 and 7.5 could disclose sensitive information via a local service to a privileged user. IBM X-Force ID: 227366."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": []
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "ibm-qradar-cve202230613-info-disc (227366)",
                            "refsource": "XF",
                            "tags": [],
                            "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/227366"
                        },
                        {
                            "name": "https://www.ibm.com/support/pages/node/6826693",
                            "refsource": "CONFIRM",
                            "tags": [],
                            "url": "https://www.ibm.com/support/pages/node/6826693"
                        }
                    ]
                }
            },
            "impact": {},
            "lastModifiedDate": "2022-10-07T18:15Z",
            "publishedDate": "2022-10-07T17:15Z"
        }
    }
}